src/Umbraco.Web/Models/ContentEditing/Tab.cs

```diff
@@ -1,30 +1,33 @@
-﻿using System.Collections.Generic;
-using System.Runtime.Serialization;
-
-namespace Umbraco.Web.Models.ContentEditing
-{
-    /// <summary>
-    /// Represents a tab in the UI
-    /// </summary>
-    [DataContract(Name = "tab", Namespace = "")]
-    public class Tab<T>
-    {
-        [DataMember(Name = "id")]
-        public int Id { get; set; }
-
-        [DataMember(Name = "active")]
-        public bool IsActive { get; set; }
-
-        [DataMember(Name = "label")]
-        public string Label { get; set; }
-
-        [DataMember(Name = "alias")]
-        public string Alias { get; set; }
-
-        [DataMember(Name = "properties")]
-        public IEnumerable<T> Properties { get; set; }
-
-        [DataMember(Name = "cssClass")]
-        public string CssClass { get; set; }
-    }
-}
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Umbraco.Web.Models.ContentEditing
+{
+    /// <summary>
+    /// Represents a tab in the UI
+    /// </summary>
+    [DataContract(Name = "tab", Namespace = "")]
+    public class Tab<T>
+    {
+        [DataMember(Name = "id")]
+        public int Id { get; set; }
+
+        [DataMember(Name = "active")]
+        public bool IsActive { get; set; }
+
+        [DataMember(Name = "label")]
+        public string Label { get; set; }
+
+        [DataMember(Name = "alias")]
+        public string Alias { get; set; }
+
+        [DataMember(Name = "properties")]
+        public IEnumerable<T> Properties { get; set; }
+
+        [DataMember(Name = "cssClass")]
+        public string CssClass { get; set; }
+
+        [DataMember(Name = "tooltip")]
+        public string Tooltip { get; set; }
```

Comment:
Unfortunately this breaks the auto mapper unit test because this is a non-mapped property during mapping operations. An explicit Ignore needs to be added in the Umbraco.Core.Models.PropertyGroup -> Umbraco.Web.Models.ContentEditing.Tab mapping

Reply:
Fixed here https://github.com/umbraco/Umbraco-CMS/commit/ffd08d0a2ef259f0772da32577d4a7b7232da75b

Reply:
Thank you for the info @Shazwazza :-)

Reply:
Thanks @Shazwazza ! Will check build failures better from now on!

Reply:
@Shazwazza Awesome 👍 I wasn't aware of that for my PR either :worried: